build_projects/dotnet-cli-build/DotNetRestore.cs

```diff
@@ -50,5 +50,10 @@ private string GetVerbosity()
 
             return null;
         }
+
+        private string GetDisableParallel()
+        {
+            return "--disable-parallel";
+        }
```

Comment:
Looking for input: what's the best way to add this in a way that it doesn't happen on Windows, where it isn't needed? Adding an extra property would work, but it's tedious in msbuild--don't know if there's another convention in CLI.